app/models/application_draft.rb

```diff
@@ -20,20 +44,21 @@ class ApplicationDraft < ActiveRecord::Base
 
   validates :team, presence: true
   validates :project1, :project_plan, presence: true, on: :apply
-  validates :heard_about_it, presence: true, on: :apply
   validates :working_together, presence: true, on: :apply
   validates :why_selected_project, presence: true, on: :apply
   validates :voluntary_hours_per_week, presence: true, on: :apply, if: :voluntary?
   validate :only_one_application_draft_allowed, if: :team, on: :create
   validate :different_projects_required
   validate :accepted_projects_required, on: :apply
+  validate :at_least_one_heard_about_it, on: :apply
 
   validates *STUDENT0_REQUIRED_FIELDS, presence: true, on: :apply
   validates *STUDENT1_REQUIRED_FIELDS, presence: true, on: :apply
   validates *STUDENT0_CHAR_LIMITED_FIELDS, length: { maximum: Student::CHARACTER_LIMIT }, on: :apply
   validates *STUDENT1_CHAR_LIMITED_FIELDS, length: { maximum: Student::CHARACTER_LIMIT }, on: :apply
 
   before_validation :set_current_season
+  before_save :clean_up_heard_about_it
```

Comment:
You should be able to get the same behavior with
```ruby
validate : heard_about_it, presence: true
```
Since you already set the array type on db level. So you could spare both, the before-save callback and the custom validation.

I'd just add a test for it and give it a try.